pgoapi/pgoapi.py

```diff
@@ -541,11 +541,22 @@ def cleanup_pokemon(self, inventory_items=None):
         caught_pokemon = self.get_caught_pokemons(inventory_items)
         for pokemons in caught_pokemon.values():
             if len(pokemons) > self.MIN_SIMILAR_POKEMON:
-                # highest lvl pokemon first
-                sorted_pokemons = sorted(pokemons, key=self.pokemon_lvl, reverse=True)
-                for pokemon in sorted_pokemons[self.MIN_SIMILAR_POKEMON:]:
-                    if self.is_pokemon_eligible_for_transfer(pokemon, sorted_pokemons[0]):
-                        self.do_release_pokemon(pokemon)
+                if self.RELEASE_DUPLICATES:
+                    sorted_pokemons = sorted(pokemons, key=self.pokemon_lvl, reverse=True)
+
+                    for pokemon in sorted_pokemons[self.MIN_SIMILAR_POKEMON:]:
+                        if self.is_pokemon_eligible_for_transfer(pokemon, sorted_pokemons[0]):
+                            self.do_release_pokemon(pokemon)
+                else:
+                    remaining = len(pokemons)
```

Comment:
I've left the initial code as is in #188, but your suggestion is perfect for when RELEASE_DUPLICATES is false.